vscode-dotnet-runtime-extension/src/extension.ts

```diff
@@ -253,6 +253,13 @@ export function activate(vsCodeContext: vscode.ExtensionContext, extensionContex
                 }
             }
 
+            // If a fully specified version (e.g., 8.0.19) is requested and forceUpdate is undefined,
+            // set forceUpdate to true to skip the existing installation check and install the specific version requested.
+            if (commandContext.version.split('.').length > 2 && commandContext.forceUpdate === undefined)
+            {
+                commandContext.forceUpdate = true;
+            }

```

Comment:
Not sure if this applies in TS, but generally, predicates and booleans are the same in a language. Meaning, you could just assign the predicate directly to `forceUpdate`. Wrapping it in an if-statement does almost nothing.